Tests/HarmonyTests/PutNetworkDataSourceTests.swift

```diff
@@ -0,0 +1,394 @@
+//
+//  PutNetworkDataSourceTests.swift
+//  HarmonyTests
+//
+//  Created by Borja Arias Drake on 11.10.2022..
+//
+
+import Alamofire
+import Foundation
+import Harmony
+import Nimble
+import XCTest
+
+@available(iOS 13.0, *)
+final class PutNetworkDataSourceTests: XCTestCase {
+
+    private typealias Utils = GenericDataSourceUtils
+    
+    func test_putAll_allobjects_query_not_supported() {
+        // Given
+        let dataSource: PutNetworkDataSource<Entity> = providePutDataSource(url: "")
+        let query = AllObjectsQuery()
+        
+        // Then
+        expectPutError(dataSource, query, CoreError.QueryNotSupported(), .putAll)
+    }
+    
+    func test_putall_returns_no_response() {
+        // Given
+        let array = [Entity(name: "a", owner: "a"), Entity(name: "b", owner: "b")]
+        let query = NetworkQuery(method: .post(type: .Json(entity: array)), path: "")
+        let url = "www.dummy.com"
```

Comment:
You should have an `anyURL()` function which returns an `URL` not just an `String`

Reply:
done